playgame.py

```diff
@@ -59,6 +78,10 @@ def run_one_game(args):
 def prep_logger(log_dir, verbose, log_stderr, count):
     formatter = logging.Formatter('[%(asctime)s] %(levelname)8s --- %(message)s ' +
                                   '(%(filename)s:%(lineno)s)',datefmt='%Y-%m-%d %H:%M:%S')
+
+    create_dirs("results",log_dir)
```

Comment:
you will only be making directories for log_dir and log_stderr if they were provided. Separate these calls in the if blocks for log_dir and log_stderr respectively